skimage/measure/profile.py

```diff
@@ -50,17 +53,27 @@ def profile_line(img, src, dst, linewidth=1,
     The destination point is included in the profile, in contrast to
     standard numpy indexing.
     """
-    perp_lines = _line_profile_coordinates(src, dst, linewidth=linewidth)
-    if img.ndim == 3:
-        pixels = [ndi.map_coordinates(img[..., i], perp_lines,
-                                      order=order, mode=mode, cval=cval)
-                  for i in range(img.shape[2])]
+    if img.ndim == 4 and multichannel:
+        # 3D RGB to be implemented
+        raise NotImplementedError('3D RGB to be implemented')
```

Comment:
This should be trivial to implement following the example above? (Although you would have to stack the channels accordingly.)